### DISCUSSION TITLE
Manually change playlist order

### DISCUSSION THREAD

**Discussion by maszek** [fast4x/RiMusic#2096] [Ideas]
Would it be possible to add a feature to change the order of a playlist manually? I know I feel kind of spoiled here, but the current order options are too limited for me:

![WhatsApp Image 2024-05-30 at 14 13 07](https://github.com/fast4x/RiMusic/assets/990963/751b3ad4-2d59-4a62-b1cf-ce8abc1fc691)

**Comment by HiroDane** [comment]
It is possible already, but its extremely hidden. So first you need to long hold the sorting arrow, that brings up a new button with "lock" icon, then you click that to "unlock" the playlist. It then gives you the handles to change order of the songs. 

But by the look of things you already have the "lock" icon. Right beside "pin" button so just unlocking that should be good.

**Reply by maszek** [reply]
Hello @HiroDane , thank you for the advice!
I tried following your instruction, but on my device/app the tracks keep jumping back to their original positions, or end up at random places in the playlist when I let go of them. 
There seems to be some room for improvement...

**Reply by fast4x** [reply]
I'll check it

**Reply by varuosb** [reply]
> I'll check it

![Screenshot_20241103-221444.png](https://github.com/user-attachments/assets/85ee0483-ec67-424c-ac64-4441f6f08ab8)

I think this is why the playlist can't rearrange check it
